src/_includes/layouts/listing-blog-post-categories.njk

```diff
@@ -38,15 +38,69 @@
 }
 ---
 
-<h1>{{ title }} "{{ tag }}"</h1>
-
 {% set blogPostLocaleTag = locale + '-blog-post' %}
+{% set articleLocaleTag = locale + '-article' %}
+{% set blogPostFilters = collections[blogPostLocaleTag] | collectionTags([blogPostLocaleTag, articleLocaleTag]) %}
 {% set blogPostCategories = collections[tag] | collectionIncludesTag(blogPostLocaleTag) | reverse %}
 
-<ul class="ul grid grid--cols-4">
-  {% for item in blogPostCategories %}
-    <li>
-      {% include "components/blog-post-card.njk" %}
-    </li>
-  {% endfor %}
-</ul>
\ No newline at end of file
+<section class="section">
+  <div class="grid grid--align-center lg:grid--cols-auto-160 mb-16 xl:mb-24">
+    <h1 class="h1 mb-0">{{ title }}</h1>
+    <div class="grid md:grid--cols-2">
+      <div>
+      <select class="select w-full" id="filters" onchange="javascript:location.href = this.value;">
+        <option value="">{{ 'category' | i18n }}</option>
```

Comment:
```suggestion
        <option value="/{{ locale }}/news/blog">{{ 'category' | i18n }}</option>
```

Could we add the blog's root URL here, so we can effectively "remove" the category filter and move back to the full listing via this control?

Reply:
@adamduncan Yeah, I think that would work.